main.css

```diff
@@ -0,0 +1,223 @@
+
+/** Breakpoints:
+ * md > 768px
+ * lg > 992px
+ */
+
+/**
+ * Main theme
+ * ======================
+ **/
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 992px) {
+  body {
+    font-size: 18px;
+  }
+}
+
+.t-bg-red {
+  background-color: #f72211;
+}
+
+.t-bg-gray {
+  background-color: #f4f5f6;
+}
+
+.t-bg-white {
+  background-color: white;
+}
+
+.t-color-red {
+  color: #f72211;
+  fill: #f72211;
+}
+
+.t-color-white {
+  color: white;
+}
+
+.t-color-black {
+  color: black;
+}
+
+.t-color-gray {
+  color: #807f84;
+}
+
+.t-color-darkgray {
+  color: #666;
+}
+
+.t-font-roboto {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.t-btn {
+  color: black;
+  font-weight: 700;
+  font-size: 13px;
+  background: #fee400;
+  border: none;
+  border-radius: 15px;
+  padding-left: 0.75rem;
+  padding-right: 0.5rem;
+  text-transform: uppercase;
+  letter-spacing: 0.08em;
+  display: flex;
+}
+
+.t-btn-icon {
+  padding-left: 4px;
+}
+
+/**
+ * Topbar
+ * ======================
+ **/
+.topbar-logo {
+  fill: white;
+  width: 106px;
+}
+
+.topbar-logo svg {
+  max-width: 100%;
+}
+
+.topbar-icon {
+  font-size: 1.5rem;
+}
+
+.topbar-login {
+  text-transform: uppercase;
+  font-weight: 700;
+  cursor: pointer;
+}
+
+@media (min-width: 992px) {
+  .topbar-logo {
+    width: 396px;
+  }
+}
+
+/**
+ * Page navigation
+ * ======================
+ **/
+.pagenav-item {
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  border-right: 1px solid #d9d9d9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] This can be done with the bootstrap classes `d-flex`, `align-items-center` and `justify-content-center`